source/binary/index.rst

```diff
@@ -135,11 +127,13 @@ Opérations
 ~~~~~~~~~~
 
 Un tableau résumant les opérations standards sur les structures se trouve sur la documentation officielle. :py:mod:`struct`.
-
+	
 Sources
 -------
 
-bytes / bytearray
+:py:class:`bytes` / :py:class:`bytearray`
+
+`<https://docs.python.org/3.1/library/stdtypes.html>`_
```

Comment:
Built-in Types est une classe ou un module?
J'ai essayé intersphinx en changeant à chaque fois mod et class avec les noms suivant:
_bltin-types
bltin-types
stdtypes
built-in types 

à chaque fois la cible n'est pas trouvée.

J'ai également essayé avec la "nouvelle" syntaxe par exemple: 
:ref:`built-in types <python:built-in types>`

Mais rien a faire...